exercises/06-deploy-to-cf.md

```diff
@@ -1,87 +1,103 @@
 # Deploy your application to Cloud Foundry
+
 Now, that you made some progress locally, let's make this application available in the cloud.
 
 ## Login to Cloud Foundry
-Login to Cloud Foundry using the following command on the command line (if you haven't logged in before):
+
+You need to login to Cloud Foundry in SAP Business Technology Platform using the cf CLI. First, you need to set an API endpoint. The exact URL of this API endpoint depends on the region your `subaccount` is in. Open the [SAP Business Technology Platform Cockpit](https://account.hana.ondemand.com/) and navigate to the subaccount you are planning to deploy your application. Click on `Overview` on the left and you can see the URL of the API endpoint.
```

Comment:
```suggestion
You need to login to Cloud Foundry in SAP Business Technology Platform using the Cloud Foundry CLI. First, you need to set an API endpoint. The exact URL of this API endpoint depends on the region your `subaccount` is in. Open the [SAP Business Technology Platform Cockpit](https://account.hana.ondemand.com/) and navigate to the subaccount you are planning to deploy your application. Click on `Overview` on the left and you can see the URL of the API endpoint.
```

see https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-ordinary-font-for-software-products